PyDSS/pydss_fs_interface.py

```diff
@@ -160,13 +163,13 @@ def _list_scenario_names(self):
 
 class PyDssDirectoryInterface(PyDssFileSystemInterface):
     """Reads PyDSS files when the project is expanded into directories."""
-    def __init__(self, project_dir):
+    def __init__(self, project_dir, scenario):
```

Comment:
Should this also be changed to `simulation_file`?

Reply:
updated